pkgs/applications/graphics/ImageMagick/7.0.nix

```
@@ -50,8 +52,12 @@ stdenv.mkDerivation rec {
 
   buildInputs =
     [ zlib fontconfig freetype ghostscript
-      libjxl liblqr1 libpng libtiff libxml2 libheif djvulibre
+      liblqr1 libpng libtiff libxml2 libheif djvulibre
     ]
+    # libjxl is broken on aarch64 (see meta.broken in libjxl) for now,
+    # let's disable it for now to unbreak the imagemagick build.
+    ++ lib.optionals (!stdenv.hostPlatform.isAarch64)
+      [ libjxl ]
```

```suggestion
    ++ lib.optionals (!stdenv.hostPlatform.isAarch64) [ libjxl ]
```

yes, all the other occurrences of this are also wrong but you did not touch them.

---

Thanks! Yeah, I guess it confused me :)